mm-core/Android.mk

```diff
@@ -121,7 +121,7 @@ LOCAL_SRC_FILES         += src/common/qc_omx_core.c
 ifneq (,$(filter msm8916 msm8994 msm8909 msm8937 msm8996 msm8992 msm8952 msm8953 msm8998 apq8098_latv sdm660,$(TARGET_BOARD_PLATFORM)))
 LOCAL_SRC_FILES         += src/$(MM_CORE_TARGET)/registry_table_android.c
 else
-LOCAL_SRC_FILES         += src/$(MM_CORE_TARGET)/qc_registry_table_android.c
+$(error "sdm660-libion media HAL: You forgot to include a new device/platform into the TARGET_BOARD_PLATFORM whitelist... Ask Felix, Martin, Marijn or Paul to fix this")
```

Comment:
In general error message should be neutral. The person that gets the message might have something forgotton but might also be just a tester / user that reads this error. Additionally naming people in an error message is not best practice. It should be avoided for several reasons.

Reply:
New error message should fix all of that.